zap/src/parser/convert.rs

```diff
@@ -92,6 +92,7 @@ impl<'src> Converter<'src> {
 
 		let (server_output, ..) = self.str_opt("server_output", "network/server.lua", &config.opts);
 		let (client_output, ..) = self.str_opt("client_output", "network/client.lua", &config.opts);
+		let (remote_scope, ..) = self.str_opt("remote_scope", "ZAP", &config.opts);
```

Comment:
Add this option (and if you could do me a favour and reconcille any missing ones) to `docs/.vitepress/components/Editor.vue` on L117.